docs/reference/api/docker_remote_api.md

```diff
@@ -99,6 +99,8 @@ This section lists each version from latest to oldest.  Each listing includes a
 * `GET /info` Now returns `Architecture` and `OSType` fields, providing information
   about the host architecture and operating system type that the daemon runs on.
 * `GET /networks/(name)` now returns a `Name` field for each container attached to the network.
+* `GET /version` now returns the `BuildTime` field in RFC3339Nano format to make it 
+  consistent with other date/time values returned by the API.
```

Comment:
well, we use the same format in all other places. might be worth describing it in general (e.g. "date/time fields in the API use the `RFC3339` standard", but that's something for another PR